plugins/inputs/mongodb/README.md

```diff
@@ -57,6 +57,7 @@ and create a single measurement containing values e.g.
  * ttl_passes_per_sec
  * repl_lag
  * jumbo_chunks (only if mongos or mongo config)
+ * oplog_timediff
```

Comment:
I think we should call the field `repl_oplog_window_s`.  This more closely matches the linked documentation from mongodb, while still using the same repl prefix as the other replication metrics and also includes the units.  I sort of dislike `_s` because its not immediately clear what it means, but it matches more in style with existing `_ns` readings.

Reply:
Okay, I will rename oplog_timediff to repl_oplog_window_s and move it closer to other repl_* metrics in code.